docs/Guides/Advertising/accountStats.md

```diff
@@ -0,0 +1,229 @@
+        "clickThroughRate": 0.0265,
+        "currencyCode": "USD",
+        "provider": "google"
+      }
+    },
+    {
+      "type": "accountStats",
+      "id": "AG-X5FZQG6T25:google:1324354698",
+      "attributes": {
+        "impressions": 5136,
+        "clicks": 99,
+        "conversions": 2,
+        "costMicros": 53000000,
+        "averageCostPerClickMicros": 535353,
+        "clickThroughRate": 0.019275,
+        "currencyCode": "CAD",
+        "provider": "google"
+      }
+    },
+    {
+      "type": "accountStats",
+      "id": "AG-X5FZQG6T25:facebook:act_63184543",
+      "attributes": {
+        "impressions": 5145,
+        "clicks": 66,
+        "conversions": 0,
+        "costMicros": 47000000,
+        "averageCostPerClickMicros": 712121,
+        "clickThroughRate": 0.012828,
+        "currencyCode": "CAD",
+        "provider": "facebook"
+      }
+    }
+  ]
+}
+```
+<!--
+type: tab-end
+-->
+
+### Filtering Stats By Date and Provider
+
+By default, this endpoint will return stats for all time.  If you want to get account stats for a specific date range, you can set `startAt` and `endAt` parameters.
+
+Similarly, if you want to get stats for only one provider, you can set the `filter[provider]` parameter to either `google`, `facebook` or `localAds`.
+
+This example will get stats from `2021-01-01` to `2021-02-01` for all Google Ads accounts connected to the businesses `AG-JT87JWQ7KN` and `AG-X5FZQG6T25`.
+
+<!--
+type: tab
+title: Request
+-->
+```json http
+{
+  "method": "get",
+  "url": "https://prod.apigateway.co/products/adintel/accountStats",
+  "headers": {
+    "Authorization": "Bearer <Access Token with 'adintel' scope>",
+    "Content-Type": "application/vnd.api+json"
+  },
+  "query": {
+    "filter[businessLocation.id]": "AG-JT87JWQ7KN,AG-X5FZQG6T25",
+    "filter[provider]": "google",
+    "startAt": "2021-01-01T00:00:00Z",
+    "endAt": "2021-02-01T00:00:00Z"
+  }
+}
+```
+
+For more details on this endpoint see [List Account Stats](../../../openapi/adintel/adintel.yaml/paths/~1accountStats/get)
+<!--
+type: tab
+title: Example Response
+-->
+```json
+{
+  "data": [
+    {
+      "type": "accountStats",
+      "id": "AG-JT87JWQ7KN:google:1348315432",
+      "attributes": {
+        "impressions": 1654,
+        "clicks": 52,
+        "conversions": 2,
+        "costMicros": 242350000,
+        "averageCostPerClickMicros": 4660577,
+        "clickThroughRate": 0.0314,
+        "currencyCode": "USD",
+        "provider": "google"
+      }
+    },
+    {
+      "type": "accountStats",
+      "id": "AG-X5FZQG6T25:google:1324354698",
+      "attributes": {
+        "impressions": 1654,
+        "clicks": 52,
+        "conversions": 2,
+        "costMicros": 242350000,
+        "averageCostPerClickMicros": 4660577,
+        "clickThroughRate": 0.0314,
+        "currencyCode": "CAD",
+        "provider": "google"
+      }
+    }
+  ]
+}
+```
+<!--
+type: tab-end
+-->
```

Comment:
Should we add an example for the resource id filter?

Reply:
I think if we do add a guide for that, it should be after the account info endpoints are in place, as the intended workflow for that would be to first get account info, then use the resource IDs of the connected accounts to get account stats.